src/AbstractFixer.php

```diff
@@ -45,6 +46,13 @@ public function __construct()
         }
     }
 
+    public function fix(\SplFileInfo $file, Tokens $tokens)
```

Comment:
If it is internal and [this](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/doc/COOKBOOK-FIXERS.md#step-1---creating-files), https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2670